tokio-console/src/warnings.rs

```diff
@@ -148,6 +148,10 @@ impl Warn<Task> for SelfWakePercent {
     }
 
     fn check(&self, task: &Task) -> Warning {
+        // Don't fire warning for tasks that are not async
+        if task.kind() == "block_on" || task.kind() == "blocking" {
+            return Warning::Ok
+        }
```

Comment:
nit, take it or leave it: this seems like it could potentially be factored out into a method on `Task`, so that we could instead just write:
```suggestion
        // Don't fire warning for tasks that are not async
        if task.is_blocking() {
            return Warning::Ok
        }
```